### PR TITLE
Update chat sidebar

### PR DESCRIPTION
**Pull request by ValentinaHotsa**

### CODE REVIEW

File: src/components/AllUsersModal/AllUsersModal.jsx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  LetterAvatarInUserBlock,
+} from './AllUsersModalStyled';
+
+const AllUsersModal = ({ isOpen, onClose }) => {
+  const currentUserId = useSelector(getUser)?.id;
+  const [searchedValue, setSearchedValue] = useState('');
+  const [filteredUsers, setFilteredUsers] = useState([]);
+
+  const { dataUserChats, updateUserChats } = useChatContext();
+  const { responseData: users } = useFetch(URLs.getAllUsers, '');
```
Comment (ReginaVarybrus) [inline src/components/AllUsersModal/AllUsersModal.jsx]: It’s unnecessary anymore to put quotation marks after URL.

File: src/components/AllUsersModal/AllUsersModalStyled.js [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+  transform: translate(-50%, -50%);
+  background: var(--white-color);
+  border-radius: 16px;
+  box-shadow:
+    0px 2px 6px 2px rgba(0, 0, 0, 0.15),
+    0px 1px 2px 0px rgba(0, 0, 0, 0.3);
+  padding: 16px;
+  width: 395px;
+
+  @media screen and (min-width: 768px) {
```
Comment (ReginaVarybrus) [inline src/components/AllUsersModal/AllUsersModalStyled.js]: Let’s use our MediaQueries.js

**Review by ReginaVarybrus — CHANGES_REQUESTED** [review]

File: src/components/AllUsersModal/AllUsersModal.jsx (hunk 1 above)
Comment (ValentinaHotsa) [reply src/components/AllUsersModal/AllUsersModal.jsx]: done

File: src/components/AllUsersModal/AllUsersModalStyled.js (hunk 2 above)
Comment (ValentinaHotsa) [reply src/components/AllUsersModal/AllUsersModalStyled.js]: done

**Review by ReginaVarybrus — APPROVED** [review]

**Pull request merged**